ArduPlane/ReleaseNotes.txt

```diff
@@ -1,5 +1,14 @@
 ArduPilot Plane Release Notes:
 ------------------------------------------------------------------
+Release 4.6.0 21-May-2025
+
+Changes from 4.6.0-beta6
+
+1) Bug Fixes
+
+- Firmware not erased on ECC error on boards using parameter area for storage
```

Comment:
should say only the TBS board changed

Reply:
Thanks for catching that, release notes fixed!